time timestamp
time timestampImage
float64 score
int32 objID
float64[3] position
float64[9] orientation
int32[2] uLCornerBB
int32[2] lRCornerBB
